lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -283,6 +283,10 @@ def setUp(self):
             npts = cube.shape[i_dim]
             coord = DimCoord(np.arange(npts), long_name=name)
             cube.add_dim_coord(coord, i_dim)
+        self.ancillary_variable = AncillaryVariable([0, 1], long_name="foo")
```

Comment:
I think adding this to a testclass called specifically `Test_collapsed__lazy` is a bit cheeky !
The actual (pre-existing) testcases show that this class is _specifically_ to test operation with lazy data.

So I think we should instead add a new testclass to be the home for these new tests.
As we don't have a "Test_collapsed", and we're not even aiming to do that here+now, I would just call it something like 'Test_collapsed__cellmeasure_ancils` + keep it minimal for this purpose.

By contrast, I think the changes to rolling-window and aggregated-by classes below are fine as they are.